cluster/aws/util.sh

```diff
@@ -1384,6 +1397,8 @@ function start-minions() {
     fi
   ) > "${KUBE_TEMP}/node-user-data"
 
+  cp ${KUBE_TEMP}/node-user-data /tmp/node-user-data
```

Comment:
what is the purpose for this line?

Reply:
$KUBE_TEMP is a random name that Jenkins has no idea. So this is to copy the file to tmp folder so Jenkins has access.